src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
I am not sure do we need it here.... I dont have it. works fine

Reply:
yes, interfaces do not need this annotation, because it is used to indicate that this **class** is an implementation that is suitable to be used in our injector, so we can remove it, from all interfaces